docs/tutorials/notebooks/sparse_monge_displacements.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    output = sinkhorn.Sinkhorn()(linear_problem.LinearProblem(geom))
</pre><p>maybe worth jitting one solver for the entire NB, and reusing it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/374/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>